site/docs/messaging/campaign-management/csp/campaign-api.mdx

```diff
@@ -100,13 +100,18 @@ HTTP/1.1 429 Too Many Requests
 | `SubscriberOptIn`          | No        | 	Does campaign require subscriber to opt-in before SMS is sent to subscriber? Default false  |
 | `SubscriberOptOut`         | No        | 	Does campaign support subscriber opt-out keyword(s)? Default false |
 | `SubscriberHelp`           | No        | 	Have you implemented a response to the HELP keyword informing customers of how they can contact the message sender? Default false   |
-| `Sample1`                  | Yes       | 	Message sample. Some campaign tiers require 1 or more message samples. Max length 1024  |
-| `Sample2`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 2 or more message samples. Max length 1024    |
-| `Sample3`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 3 or more message samples. Max length 1024   |
-| `Sample4`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 4 or more message samples. Max length 1024   |
-| `Sample5`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 5 message samples. Max length 1024  |
-| `MessageFlow`              | No        | 	Message flow description. Max length 2048 |
-| `HelpMessage`              | Yes       | 	Help message of the campaign. Max length 255  |
+| `Sample1`                  | Yes       | 	Message sample. Some campaign tiers require 1 or more message samples. Min length 20. Max length 1024  |
+| `Sample2`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 2 or more message samples. Min length 20. Max length 1024    |
+| `Sample3`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 3 or more message samples. Min length 20. Max length 1024   |
+| `Sample4`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 4 or more message samples. Min length 20. Max length 1024   |
+| `Sample5`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 5 message samples. Min length 20. Max length 1024  |
+| `MessageFlow`              | Yes       | 	Message flow description. Min length 40. Max length 2048 |
+| `HelpMessage`              | Yes       | 	Help message of the campaign. Min length 20. Max length 320  |
+| `HelpKeywords`             | No        | 	Help keywords of the campaign. Max length 320. TCR will default the value to HELP  |
+| `OptinMessage`             | No        | 	Opt-In message of the campaign. Min length 20. Max length 320  |
+| `OptinKeywords`            | No        | 	Opt-In keywords of the campaign. Max length 320.  |
```

Comment:
Opt-In has no default keywords, like OptOutKeywords does?

Reply:
Poor design by TCR so unfortunately not. Here's the doc: chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://www.campaignregistry.com/Additional-Data-Capture.pdf?utm_medium=email&_hsmi=230706115&_hsenc=p2ANqtz-9BKBTTyG9_OnYX9s4zE3Irlp46dmCE7A2RRt7jvg5QImweOuSZJV2uYQcGJKZn02etDJUmYe0bfUzzKuiLdDalJO_jmg&utm_content=230706115&utm_source=hs_email